client/components/button/style.scss

```diff
@@ -76,6 +76,14 @@ button {
 			top: 4px;
 			margin-top: -8px;
 		}
+		// Make the left margin of the small plus icon visually less huge
+		.gridicons-plus-small {
```

Comment:
Will there ever be a condition in which we'd only use the `+` icon? If so, would this make that icon be off-center?